pytorch_lightning/utilities/enums.py

```diff
@@ -94,3 +94,10 @@ class DeviceType(LightningEnum):
     CPU = 'CPU'
     GPU = 'GPU'
     TPU = 'TPU'
+
+
+class GradClipAlgorithmType(LightningEnum):
+    """ Define gradient_clip_algorithm types - training-tricks.
+    """
```

Comment:
Can you expand this docstring with the difference between types?

Reply:
Okay I will.

Reply:
Updated :)